src/main/java/com/exasol/releasedroid/usecases/release/ReleaseInteractor.java

```diff
@@ -18,6 +21,7 @@
     private final ValidateUseCase validateUseCase;
     private final Map<PlatformName, ? extends ReleaseMaker> releaseMakers;
     private final RepositoryGateway repositoryGateway;
+    private final List<PlatformName> releasePriority = List.of(MAVEN, GITHUB);
```

Comment:
this should be moved to the constructor. Remember the business rules have no idea about `Maven` or `Github`